docs/getting_started.md

```diff
@@ -50,6 +50,8 @@ It is required that machines provisioned by CAPV use one of the official CAPV ma
 
 [Create a VM template](https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vm_admin.doc/GUID-17BEDA21-43F6-41F4-8FB2-E01D275FE9B4.html) using the OVA URL above. The rest of the guide will assume you named the VM template `ubuntu-1804-kube-v1.15.4`.
 
+**Note:** When creating OVA template via vSphere using URL method, make sure VM template name is same as value specified for environment variable VSPHERE_TEMPLATE in envvars.txt file, taking care of .ova suffix for the template name.
```

Comment:
Hi @shivaani0505,

A few notes:

* Please use the appropriate articles:
  * _When creating OVA_ should be _When creating the OVA_
  * _using URL method_ should be _using the URL method_
  * _make sure VM template_ should be _make sure the VM template_
  * _name is same_ should be _name is the same_
  * _value specified_ should be _the value specified_
  * _specified for_ should be _specified by_
  * _environment variable_ should be _the environment variable_
  * _in envvars.txt file_ should be _in the `envvars.txt` file_
  * _taking care of .ova suffix_ should be _taking care of the `.ova` suffix_
* Please use proper formatting when appropriate. For example:
  * VSPHERE_TEMPLATE should be `VSPHERE_TEMPLATE`
  * envvars.txt should be `envvars.txt`
  * .ova should be `.ova`
* I like to add _please_ whenever it makes sense, such as _, make sure_ could become _, please make sure_